client/src/app.jsx

```diff
@@ -1,14 +1,19 @@
 import React from 'react';
 
 import ProductOverview from './Components/ProductOverview/ProductOverview.jsx';
-import QuestionAnswer from './Components/QuestionAnswer/QuestionAnswer.jsx';
+import RelatedProductsView from './Components/RelatedProducts/RelatedProductsView.jsx';
```

Comment:
Should merge conflicts in this file and index.jsx file to be resolved locally and then do pull request?